### DISCUSSION TITLE
When or how the latest BIOS/firmware version come into fwupd?

### DISCUSSION THREAD

**Discussion by rostidev** [fwupd/fwupd#8063] [Q&A]
I use Fedora 41 with fwupd 1.9.26 on HP EliteBook 840 G11

The latest version of the BIOS firmware of this notebook is [**01.03.05**](https://support.hp.com/ie-en/drivers/swdetails/hp-elitebook-840-14-inch-g11-notebook-pc/model/2102057964/swItemId/ob-337095-1?sku=9G0C5ET) released in September 18. This version still isn't known for fwupd, even after `fwupdmgr refresh --force`. When or how that BIOS firmware version come into fwupd? Or how can I extract it by myself from the `sp155139.exe` file provided by HP for BIOS firmware update on Window?

**Comment by hughsie** [comment]
You need to ask HP to add the latest firmware to the LVFS I'm afraid. See https://fwupd.org/lvfs/devices/ for the supported devices.
